ccpp/data/GFS_typedefs.F90

```diff
@@ -3053,7 +3051,7 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
                                  nwat, tracer_names, tracer_types,  &
                                  input_nml_file, tile_num, blksz,   &
                                  ak, bk, restart, hydrostatic,      &
-                                 communicator, ntasks, nthreads)
+                                 communicator, ntasks, nthreads )
```

Comment:
Don't think this change is necessary?

Reply:
Hmm, I reverted that change.  Guess I need to do it again.